src/app/components/AllowanceMenu/index.test.tsx

```diff
@@ -1,10 +1,22 @@
 import { act, render, screen, waitFor } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
 import { MemoryRouter } from "react-router-dom";
+import * as SettingsContext from "~/app/context/SettingsContext";
+import type { SettingsStorage } from "~/types";
 
 import type { Props } from "./index";
 import AllowanceMenu from "./index";
 
+const mockSettings = {
+  showFiat: true,
+} as SettingsStorage;
```

Comment:
Would be nice if we could avoid `as`. Would it work to define all needed props and do `mockSettings:SettingsStorage = {`? Same for the other usages.

Reply:
Changes everywhere with adding a new fixtures folder for tests (=> see also the discussion [here](https://github.com/getAlby/lightning-browser-extension/pull/968#discussion_r928803305))